legal-api/report-templates/certificateOfDissolution.html

```diff
@@ -0,0 +1,67 @@
+[[macros.html]]
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Certificate of Dissolution</title>
+    <meta charset="UTF-8">
+    <meta name="author" content="BC Registries and Online Services">
+    [[common/certificateStyle.html]]
+  </head>
+
+  <body>
+    <div class="outer-container">
+      <div class="outer-container-header">
+        <div class="business-number">Incorporation Number: {{ business.identifier }}</div>
+        [[common/correctedOnCertificate.html]]
+      </div>
+
+      <div class="certificate-statements">
+          [[common/certificateLogo.html]]
+
+          <div class="cert-title-container">
+            <div class="sub-header-1">Certificate  of</div>
```

Comment:
2 spaces between "Certificate" and "of".

Reply:
Nice catch!